.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
tego nie powinno byc w repo, dodaj to do globalnego gitignora

Reply:
Dodałem gitignora i tam jest